desktop/sources/scripts/client.js

```diff
@@ -26,6 +28,29 @@ function Client () {
   this.commander = new Commander(this)
   this.clock = new Clock(this)
 
+  // Ableton Link
+  this.link = new AbletonLink();
+
+  this.link.setTempoCallback((newTempo) => {
+    newTempo = this.link.getTempo(true)
+    if (this.clock.isLinkEnabled && this.clock.speed.value != newTempo) {
+      this.clock.setSpeed(newTempo, newTempo, true)
+      this.clock.setFrame(0)
+      this.update()
+    };
+  });
+  
+  this.link.setStartStopCallback((startStopState) => {
+    console.log("startstop: " + startStopState);
+    if (startStopState && this.clock.isPaused) {
+      this.clock.play(false, true, true)
```

Comment:
`midiStart` (second argument) should probably be `false`, it is getting ignored anyway. Same for `stop()` below.